cli/dcoscli/data/help/marathon.txt

```diff
@@ -28,6 +28,9 @@ Usage:
     dcos marathon group show [--group-version=<group-version>] <group-id>
     dcos marathon group remove [--force] <group-id>
     dcos marathon group update [--force] <group-id> [<properties>...]
+    dcos marathon leader delete
+    dcos marathon leader show [--json]
+    dcos marathon ping [--once] [--timeout=<timeout>]
```

Comment:
having timeout + once seems a little redundant. There is also a global timeout param for the CLI. I would just keep the `once` param.